### PR TITLE
Add number validation for sitemap file limits

### PR DESCRIPTION
**Pull request by ravi-chandra3197**
### Description (*)

Add number validation for sitemap file limits
### Fixed Issues (if relevant)

1. magento/magento2#<issue_number>: N/A

### Manual testing scenarios (*)

1. Go to admin area
2. Go to Stores -> Configuration -> Catalog -> XML Sitemap -> Sitemap File Limits
3. Edit Maximum No of URLs Per File and Maximum File Size
4. Save the config

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All new or changed code is covered with unit/integration tests (if applicable)
 - [x] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Review by VladimirZaets — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @VladimirZaets, thank you for the review.
[ENGCOM-5731](https://jira.corp.magento.com/browse/ENGCOM-5731) has been created to process this Pull Request
:eight_spoked_asterisk: @VladimirZaets, could you please add one of the following labels to the Pull Request?

| Label | Description |
|--------------------------------|-----------------------------------------------------------------|
| ```Auto-Tests: Covered``` | All changes in Pull Request is covered by auto-tests |
| ```Auto-Tests: Not Covered``` | Changes in Pull Request requires coverage by auto-tests |
| ```Auto-Tests: Not Required``` | Changes in Pull Request does not require coverage by auto-tests |

**Comment by engcom-Alfa** [comment]
Hi @ravi-chandra3197, we can still use negative numbers. (_-10_ for ex.) This is incorrect validation.

Could you take a look, please?

Thanks!

File: app/code/Magento/Sitemap/etc/adminhtml/system.xml [hunk 1]
```diff
@@ -83,10 +83,12 @@
                 <label>Sitemap File Limits</label>
                 <field id="max_lines" translate="label" type="text" sortOrder="1" showInDefault="1" showInWebsite="1" showInStore="1" canRestore="1">
                     <label>Maximum No of URLs Per File</label>
+                    <validate>validate-number</validate>
```
Comment (oleksii-lisovyi) [inline app/code/Magento/Sitemap/etc/adminhtml/system.xml]: There is another validation class "validate-greater-than-zero", that should fix the problem with values below or equal to zero.

**Comment by ravi-chandra3197** [comment]
> Hi @ravi-chandra3197, we can still use negative numbers. (_-10_ for ex.) This is incorrect validation.
> 
> Could you take a look, please?
> 
> Thanks!

Hello @engcom-Alfa 
I  have update PR can you check it.

File: app/code/Magento/Sitemap/etc/adminhtml/system.xml (hunk 1 above)
Comment (ravi-chandra3197) [reply app/code/Magento/Sitemap/etc/adminhtml/system.xml]: I have added validation for greater than zero

**Review by sidolov — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @sidolov, thank you for the review.
[ENGCOM-5731](https://jira.corp.magento.com/browse/ENGCOM-5731) has been created to process this Pull Request
:eight_spoked_asterisk: @sidolov, could you please add one of the following labels to the Pull Request?

| Label | Description |
|--------------------------------|-----------------------------------------------------------------|
| ```Auto-Tests: Covered``` | All changes in Pull Request is covered by auto-tests |
| ```Auto-Tests: Not Covered``` | Changes in Pull Request requires coverage by auto-tests |
| ```Auto-Tests: Not Required``` | Changes in Pull Request does not require coverage by auto-tests |

**Comment by engcom-Alfa** [comment]
:heavy_check_mark: QA Passed

**Pull request merged**